ccleaner.txt

```diff
@@ -8,6 +8,7 @@ URLSite = https://www.ccleaner.com/ccleaner
 URLDownload = https://download.ccleaner.com/ccsetup562.exe
 SHA1 = e5c348ec9ff7905cb7f09a61a8686cb1e32154de
 SizeBytes = 25441808
+Icon=ccleaner.ico
```

Comment:
I extracted the icon directly from the application to prevent it from being in an incorrect format even if it was marked as .ico.

Do you recommend compressing that ICO and what tool could I use?